src/get_data.py

```diff
@@ -80,3 +81,69 @@ def get_metric_value(
     metric_values = query.all()
 
     return metric_values
+
+
+def get_gsp_yield_sum(
```

Comment:
Think it would be better if we import this from `nowcast_datamodel` then we dont have to duplicated code

`query = query.where(GSPYieldSQL.solar_generation_kw+1 > GSPYieldSQL.solar_generation_kw)` lets add a issue on `nowcasting_datamodel` to sovle this

Reply:
I'll go ahead and create the issue for the `nowcasting_datamodel`.

Reply:
yea, would be good to grab that too, probably good to check that `get_metric_value` is the same